dist/tools/esptools/export.sh

```diff
@@ -15,6 +22,47 @@ fi
 
 TOOLS_PATH="${IDF_TOOLS_PATH}/tools"
 
+# this function expects the parameters $TOOL, $TOOLS_DIR and $*_VERSION
+export_checks()
+{
+    TOOL="$1"
+    TOOLS_DIR_INT="$2" # internal TOOLS_DIR
+    TOOLS_VERSION="$3"
+
+    # create the wildcard expression from the TOOLS_DIR
+    TOOLS_DIR_BASE=$(echo "$TOOLS_DIR_INT/bin" | sed "s|/$TOOLS_VERSION/|/[^/]*/|")
+    TOOLS_DIR_IN_PATH=$(echo "$PATH" | grep "${TOOLS_DIR_INT}")
+
+    if [ ! -e "${TOOLS_DIR_INT}" ]; then
+        echo "${TOOLS_DIR_INT} does not exist - please run"
+        echo "\${RIOTBASE}/dist/tools/esptools/install.sh $TOOL"
+        return
+    fi
+
+    echo "$PATH" | tr ':' '\n' | while read -r entry; do
+        if echo "$entry" | grep -q "^${TOOLS_DIR_BASE}$"; then
+            if [ "$entry" != "${TOOLS_DIR_INT}/bin" ]; then
+                echo "Warning: PATH contains outdated entry: \"$entry\"." \
+                     "Please check your ~/.bashrc or ~/.profile.">&2
+            fi
+	fi
+    done
+    unset entry
+
+    if [ -e "${TOOLS_DIR_INT}" ] && [ -z "${TOOLS_DIR_IN_PATH}" ]; then
+        echo "Extending PATH by ${TOOLS_DIR_INT}/bin"
+        export PATH="${TOOLS_DIR_INT}/bin:${PATH}"
+
+        echo "To make this permanent, add this line to your ~/.bashrc or ~/.profile:"
+        echo PATH="\$PATH:${TOOLS_DIR_INT}/bin"
+    fi
+
+    unset TOOL
+    unset TOOLS_DIR_INT
+    unset TOOLS_VERSION
+    unset TOOLS_DIR_IN_PATH
+}
```

Comment:
We need the return value in case of success for OpenOCD settings, see below:
```suggestion

    return 0
}
```